README.md

```diff
@@ -185,8 +185,27 @@ SET s3_region='us-west-2';
 
 Check out [example notebooks here]() for instructions on how to use DuckDB inside a notebook for a more interactive experience. -->
 
+### 4. Apache Sedona (Python + Spatial SQL)
 
-### 4. Download the Parquet files
+You can get a single-node Sedona Docker image from [Apache Software Foundation DockerHub](https://hub.docker.com/r/apache/sedona) and run `docker run -p 8888:8888 apache/sedona:latest`. A Jupyter Lab and notebook examples will be available at http://localhost:8888/. You can also install Sedona to Databricks, AWS EMR and Snowflake using [Wherobots](https://www.wherobots.ai/demo).
+
+The following Python + Spatial SQL code reads the Places dataset and runs a spatial filter query on it.
+
+```
+import sedona.spark.*
+
+config = SedonaContext.builder().getOrCreate()
```

Comment:
In order to access the files, I had to use this: 

```
config = SedonaContext.builder().config("fs.s3a.aws.credentials.provider", 
                                        "org.apache.hadoop.fs.s3a.AnonymousAWSCredentialsProvider").getOrCreate()
sedona = SedonaContext.create(config)
```

Reply:
Fixed.